app/components/login.tsx

```diff
@@ -0,0 +1,117 @@
+'use client'
+import Link from 'next/link'
+import { useState, FormEvent } from 'react'
+import Image from 'next/image'
+import profile from './profile.svg'
+export default function Login_Form() {
+  const [email, setEmail] = useState<string>('')
+  const [password, setPassword] = useState<string>('')
+
+  const handleEmailChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setEmail(event.target.value)
```

Comment:
@mohd-khan09 We can have a single state for handling these inputs, also consider creating a custom hook for inputs handling.